src/e2e.js

```diff
@@ -236,6 +235,29 @@ mye2e.prototype.importKey = function(keyStr, passphrase) {
       }, keyStr);
 };
 
+mye2e.prototype.importPubKey = function(keyStr) {
+  var pgp = this.pgpContext;
+  return pgp.getKeyDescription(keyStr).then(function(keyDescriptions) {
+    var keyDescription = keyDescriptions[0];
```

Comment:
Could you add a brief comment like "lookup existing key description to avoid collisions"?